packages/client/src/components/dataset/Dataset.tsx

```diff
@@ -5,7 +5,7 @@ import MoreVertIcon from "@material-ui/icons/MoreVert";
 import { LazyLoadImage, ScrollPosition, trackWindowScroll } from "react-lazy-load-image-component";
 import { GetDataset, GetDataset_dataset_images, GetDatasetVariables } from "./__generated__/GetDataset";
 import { useQuery } from "@apollo/client";
-import RenameDatasetDialogButton from "./RenameDatasetDialog";
+import RenameDatasetDialogButton from "./RenameProjectDialog";
```

Comment:
Was importing dataset buttons from a project file intentional?

Reply:
Has this been a thing for a while and I havn't noticed?

Reply:
filename was wrong